source_code/aux_mcu_v4/src/fido2/solo_compat_layer.c

```diff
@@ -0,0 +1,67 @@
+#include "driver_timer.h"
+#include "solo_compat_layer.h"
+#include "comms_raw_hid.h"
+#include "platform_defines.h"
+
+uint32_t millis(void)
+{
+    return timer_get_systick();
+}
+
+void usbhid_send(uint8_t * msg)
```

Comment:
to be renamed with something more appropriate

Reply:
ok